prisma/schema.prisma

```diff
@@ -132,3 +132,16 @@ model Information {
   @@unique([userId, userEmail])
   @@map("informations")
 }
+
+model Career {
+  id        String   @id @default(cuid())
+  jobTitle  String
+  company   String
+  startDate DateTime 
+  endDate   DateTime 
+  userId    String   @unique
```

Comment:
I think it should not be unique, right? unique is for 1-1 relation

Reply:
https://www.prisma.io/docs/concepts/components/prisma-schema/relations

Reply:
As I read the doc, In this case, for each User there can only be a unique Career.
 Example - (User1, Career1), (User1, Career2). Adding another record with (User1, Career1) would throw an error 

it can be used for 1 - many. I think this can be optional and used in this situation.

Reply:
`Example - (User1, Career1), (User1, Career2).`
As you see career1 and career2 have the same user.

This is 1 - many and optional
![image](https://user-images.githubusercontent.com/63942110/211959790-cee8f3a2-ecd9-477a-8f58-2696eb4ca312.png)

Reply:
Because you add @unique in `userId` in career, so each `career ` record must belong to a unique `user`. So it make this relationship become 1-1